ansible/roles/bastion-lite/files/bastion_ssh_config.j2

```diff
@@ -9,3 +10,17 @@ Host ec2* *.internal *.example.com
   StrictHostKeyChecking no
   ConnectTimeout 60
   ConnectionAttempts 10
+{% endif %}	
+{% if cloud_provider == 'osp' %}	
+Host *.example.com
+  User {{remote_user}}	
+{% if use_own_key|bool %}	
```

Comment:
Can we add some indentation in the template to help trace the nested conditions? Something like:

```
{% if cloud_provider == 'osp' %}
...
{%   if use_own_key|bool %}
...
{%   else %}
...
{%   endif %}
...
{% endif %}
```